src/Raven.Client/Documents/Operations/Attachments/Retired/DeleteRetiredAttachmentOperation.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Attachments.Retired
+{
+    public sealed class DeleteRetiredAttachmentOperation : IOperation
+    {
+        private readonly string _documentId;
+        private readonly string _name;
+        private readonly string _changeVector;
+        private readonly bool _storageOnly;
+
+        public DeleteRetiredAttachmentOperation(string documentId, string name, string changeVector = null, bool storageOnly = false)
```

Comment:
validate parameters

Reply:
maybe instead of 'storageOnly' we want sth like 'AttachmentType' ? bool is less expandable